package/kernel/linux/modules/netdevices.mk

```diff
@@ -1141,3 +1141,23 @@ define KernelPackage/mlx5-core/description
 endef
 
 $(eval $(call KernelPackage,mlx5-core))
+
+
+define KernelPackage/sfp
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=SFP cage support
+  DEPENDS:=+kmod-i2c-core
+  KCONFIG:= \
+    CONFIG_SFP=m \
+    CONFIG_MDIO_I2C=m
```

Comment:
Should this also depend on +kmod-hwmon-core and some form of KCONFIG too?

Reply:
I added kmod-hwmon-core to the DEPENDS line.